client/js/middleware/serialization/qbank/serializers/movable_word_sentence.js

```diff
@@ -56,7 +56,7 @@ function serializeAnswers(choices, newChoiceAttributes, oldAnswers, correctFeedb
     genusTypeId: genusTypes.answer.rightAnswer,
     feedback: _.get(correctFeedback, 'text'),
     type: genusTypes.answer.multipleAnswer,
-    choiceIds: _.map(_.orderBy(_.filter(updatedChoices, choice => choice.answerOrder !== ''), 'answerOrder'), 'id'),
+    choiceIds: _.map(_.orderBy(_.filter(updatedChoices, choice => !_.isNil(choice.answerOrder) && choice.answerOrder !== ''), 'answerOrder'), 'id'),
```

Comment:
with three chained things, maybe use lodash chaining? This would changed to `_(updatedChoices).filter(choice => !_.isNil(choice.answerOrder) && choice.answerOrder !== '').orderBy('answerOrder').map('id')`